2025-04-18 13:18:35,431 INFO    Thread-6 (_run_job):59793 [wandb_setup.py:_flush():67] Current SDK version is 0.19.7
2025-04-18 13:18:35,432 INFO    Thread-6 (_run_job):59793 [wandb_setup.py:_flush():67] Configure stats pid to 59793
2025-04-18 13:18:35,432 INFO    Thread-6 (_run_job):59793 [wandb_setup.py:_flush():67] Loading settings from /Users/rudra_sarkar/.config/wandb/settings
2025-04-18 13:18:35,432 INFO    Thread-6 (_run_job):59793 [wandb_setup.py:_flush():67] Loading settings from /Users/rudra_sarkar/Documents/PhD IIT Madras/Intro to Deep Learning M Khapra DA6401/da6401 assignment 2/wandb/settings
2025-04-18 13:18:35,432 INFO    Thread-6 (_run_job):59793 [wandb_setup.py:_flush():67] Loading settings from environment variables
2025-04-18 13:18:35,432 INFO    Thread-6 (_run_job):59793 [wandb_init.py:setup_run_log_directory():647] Logging user logs to /Users/rudra_sarkar/Documents/PhD IIT Madras/Intro to Deep Learning M Khapra DA6401/da6401 assignment 2/wandb/run-20250418_131835-qgz5ch47/logs/debug.log
2025-04-18 13:18:35,433 INFO    Thread-6 (_run_job):59793 [wandb_init.py:setup_run_log_directory():648] Logging internal logs to /Users/rudra_sarkar/Documents/PhD IIT Madras/Intro to Deep Learning M Khapra DA6401/da6401 assignment 2/wandb/run-20250418_131835-qgz5ch47/logs/debug-internal.log
2025-04-18 13:18:35,433 INFO    Thread-6 (_run_job):59793 [wandb_init.py:init():761] calling init triggers
2025-04-18 13:18:35,433 INFO    Thread-6 (_run_job):59793 [wandb_init.py:init():766] wandb.init called with sweep_config: {'augment': False, 'batch_norm': False, 'batch_size': 32, 'conv_activation': 'silu', 'dense': [512], 'dense_activation': 'mish', 'dropout': 0.5, 'filter_initial': 64, 'filter_strategy': 'same', 'kernel': [5, 5, 5, 5, 5], 'learning_rate': 0.0006474102879962732, 'momentum': 0.8744984220569143, 'optimizer': 'rmsprop', 'resize': 32, 'weight_decay': 0.0028323764735565192}
config: {'batch_size': 32, 'resize': 64, 'filter_automated': True, 'filter_strategy': 'halved', 'filter_initial': 16, 'n_convolutions': 5, 'filter_manual': [16, 32, 64, 128, 256], 'padding': True, 'stride': [1, 1, 1, 1, 1], 'dense': [128], 'kernel': [3, 3, 3, 3, 3], 'conv_activation': 'sigmoid', 'dense_activation': 'tanh', 'n_dense': 1, 'optimizer': 'rmsprop', 'augment': True, 'batch_norm': True, 'learning_rate': 0.0006563984074837907, 'momentum': 0.8069287304990089, 'weight_decay': 0.0014556827831633356, 'dropout': 0.5, 'xavier_init': False, 'wandb': False, 'wandb_entity': 'da24d008-iit-madras', 'wandb_project': 'da6401-assignment2', 'wandb_sweep': True, 'sweep_id': None, '_wandb': {}}
2025-04-18 13:18:35,433 INFO    Thread-6 (_run_job):59793 [wandb_init.py:init():784] starting backend
2025-04-18 13:18:35,433 INFO    Thread-6 (_run_job):59793 [wandb_init.py:init():788] sending inform_init request
2025-04-18 13:18:35,438 INFO    Thread-6 (_run_job):59793 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2025-04-18 13:18:35,439 INFO    Thread-6 (_run_job):59793 [wandb_init.py:init():803] backend started and connected
2025-04-18 13:18:35,443 INFO    Thread-6 (_run_job):59793 [wandb_run.py:_config_callback():1261] config_cb None None {'augment': False, 'batch_norm': False, 'batch_size': 32, 'conv_activation': 'silu', 'dense': [512], 'dense_activation': 'mish', 'dropout': 0.5, 'filter_initial': 64, 'filter_strategy': 'same', 'kernel': [5, 5, 5, 5, 5], 'learning_rate': 0.0006474102879962732, 'momentum': 0.8744984220569143, 'optimizer': 'rmsprop', 'resize': 32, 'weight_decay': 0.0028323764735565192}
2025-04-18 13:18:35,452 INFO    Thread-6 (_run_job):59793 [wandb_init.py:init():896] updated telemetry
2025-04-18 13:18:35,495 INFO    Thread-6 (_run_job):59793 [wandb_init.py:init():920] communicating run to backend with 90.0 second timeout
2025-04-18 13:18:35,971 INFO    Thread-6 (_run_job):59793 [wandb_init.py:init():995] starting run threads in backend
2025-04-18 13:18:36,169 INFO    Thread-6 (_run_job):59793 [wandb_run.py:_console_start():2377] atexit reg
2025-04-18 13:18:36,169 INFO    Thread-6 (_run_job):59793 [wandb_run.py:_redirect():2227] redirect: wrap_raw
2025-04-18 13:18:36,169 INFO    Thread-6 (_run_job):59793 [wandb_run.py:_redirect():2292] Wrapping output streams.
2025-04-18 13:18:36,169 INFO    Thread-6 (_run_job):59793 [wandb_run.py:_redirect():2317] Redirects installed.
2025-04-18 13:18:36,170 INFO    Thread-6 (_run_job):59793 [wandb_init.py:init():1037] run started, returning control to user process
2025-04-18 13:19:01,043 INFO    MainThread:59793 [wandb_run.py:_finish():2112] finishing run da24d008-iit-madras/da6401-assignment2/qgz5ch47
2025-04-18 13:19:01,044 INFO    MainThread:59793 [wandb_run.py:_atexit_cleanup():2342] got exitcode: 0
2025-04-18 13:19:01,044 INFO    MainThread:59793 [wandb_run.py:_restore():2324] restore
2025-04-18 13:19:01,045 INFO    MainThread:59793 [wandb_run.py:_restore():2330] restore done
2025-04-18 13:19:01,528 WARNING MsgRouterThr:59793 [router.py:message_loop():77] message_loop has been closed
Traceback (most recent call last):
  File "/opt/anaconda3/envs/nlp_env/lib/python3.12/site-packages/wandb/sdk/interface/router_sock.py", line 28, in _read_message
    resp = self._sock_client.read_server_response(timeout=1)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/nlp_env/lib/python3.12/site-packages/wandb/sdk/lib/sock_client.py", line 285, in read_server_response
    data = self._read_packet_bytes(timeout=timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/nlp_env/lib/python3.12/site-packages/wandb/sdk/lib/sock_client.py", line 270, in _read_packet_bytes
    raise SockClientClosedError
wandb.sdk.lib.sock_client.SockClientClosedError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/opt/anaconda3/envs/nlp_env/lib/python3.12/site-packages/wandb/sdk/interface/router.py", line 70, in message_loop
    msg = self._read_message()
          ^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/nlp_env/lib/python3.12/site-packages/wandb/sdk/interface/router_sock.py", line 30, in _read_message
    raise MessageRouterClosedError from e
wandb.sdk.interface.router.MessageRouterClosedError
2025-04-18 13:19:01,530 INFO    MsgRouterThr:59793 [mailbox.py:close():115] Closing mailbox, abandoning 2 handles.
